src/test/benchmarks/jetton/jetton.spec.ts

```diff
@@ -0,0 +1,462 @@
+        transfer: 26852n,
+        burn: 17728n,
+        discovery: 11055n,
+    },
+    {
+        label: "master",
+        transfer: 26568n,
+        burn: 17718n,
+        discovery: 11063n,
+    },
+];
+
+function calculateChanges(results: BenchmarkResult[]): string[][] {
+    type MetricKey = Exclude<keyof BenchmarkResult, "label">;
+    const headers = ["transfer", "burn", "discovery"] as const;
+    const changes: string[][] = results.map(() => headers.map(() => ""));
+
+    for (let i = 1; i < results.length; i++) {
+        headers.forEach((key, index) => {
+            const prevValue = results[i - 1]![key as MetricKey];
+            const currValue = results[i]![key as MetricKey];
+            const change = (
+                (Number(currValue - prevValue) / Number(prevValue)) *
+                100
+            ).toFixed(2);
+            changes[i]![index] =
+                parseFloat(change) >= 0
+                    ? `\x1b[91m+${change}%\x1b[0m`
+                    : `\x1b[32m${change}%\x1b[0m`;
+        });
+    }
+    return changes;
+}
+
+function printBenchmarkTable(results: BenchmarkResult[]): void {
+    if (results.length === 0) {
+        console.log("No benchmark results to display.");
+        return;
+    }
+
+    const table = new Table({
+        head: ["Run", "Transfer", "Burn", "Discovery"],
+        style: {
+            head: ["cyan"],
+            border: ["gray"],
+        },
+    });
+
+    const changes = calculateChanges(results);
+
+    results.forEach((result, i) => {
+        table.push([
+            result.label,
+            `${result.transfer} ${changes[i]?.[0] ?? ""}`,
+            `${result.burn} ${changes[i]?.[1] ?? ""}`,
+            `${result.discovery} ${changes[i]?.[2] ?? ""}`,
+        ]);
+    });
+
+    process.stdout.write(table.toString());
+    process.stdout.write("\n");
+
+    const first = results[0]!;
+    const last = results[results.length - 1]!;
+    const compareMetrics = ["transfer", "burn", "discovery"] as const;
+
+    process.stdout.write("\nComparison with FunC implementation:\n");
+    compareMetrics.forEach((metric) => {
+        const ratio = (
+            (Number(last[metric]) / Number(first[metric])) *
+            100
+        ).toFixed(2);
+        process.stdout.write(
+            `${metric.charAt(0).toUpperCase() + metric.slice(1)}: ${
+                parseFloat(ratio) > 100
+                    ? `\x1b[91m${ratio}%\x1b[0m`
+                    : `\x1b[32m${ratio}%\x1b[0m`
+            } of FunC gas usage\n`,
+        );
+    });
+    process.stdout.write("\n");
+}
+
+function getUsedGas(sendEnough: SendMessageResult) {
+    return sendEnough.transactions
+        .slice(1)
+        .map((t) => {
+            return (
+                (t.description as TransactionDescriptionGeneric)
+                    .computePhase as TransactionComputeVm
+            ).gasUsed;
+        })
+        .reduceRight((prev, cur) => prev + cur);
+}
```

Comment:
This stuff looks like it can be reused in all the benchmarks

Reply:
Moved to benchmark `util.ts`